db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ProjectRestrictNode.java

```diff
@@ -1189,8 +1189,15 @@ public void generateResultSet(ExpressionClassBuilder acb, MethodBuilder mb) thro
         int i = 0;
         if (genExpressions) {
             for (ResultColumn rc : resultColumns) {
-                expressions[i] = OperatorToString.opToSparkString(rc.getExpression());
-                if (expressions[i] == null) {
+                try {
+                    expressions[i] = OperatorToString.opToSparkString(rc.getExpression());
+                    if (expressions[i] == null) {
+                        genExpressions = false;
+                        numberOfValues = 0;
+                        break;
+                    }
+                }
+                catch (StandardException e) {
                     genExpressions = false;
```

Comment:
This would be an unexpected exception, we should log it at least, but I would prefer to make it bubble up. If we don't trigger it on the regression tests we can assume nothing breaks.

Reply:
> This would be an unexpected exception, we should log it at least, but I would prefer to make it bubble up. If we don't trigger it on the regression tests we can assume nothing breaks.

OK, removed the try/catches, so unexpected exceptions are caught.